crates/evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -113,12 +113,47 @@ impl ArtifactBytecode {
         }
     }
 
-    fn into_deployed_bytecode(self) -> Option<Bytes> {
+    fn into_deployed_bytecode(self, immutables: &[[u8; 32]]) -> Option<Bytes> {
```

Comment:
I don't know what this does.

this needs docs, I'd be totally lost when I have to revisit this again